integration-cli/docker_cli_events_test.go

```diff
@@ -206,18 +206,26 @@ func TestEventsImagePull(t *testing.T) {
 func TestEventsImageImport(t *testing.T) {
 	since := time.Now().Unix()
 
-	defer deleteImages("cirros")
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "true")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		t.Fatal("failed to create a container", out, err)
+	}
+	cleanedContainerID := stripTrailingCharacters(out)
+	defer deleteContainer(cleanedContainerID)
 
-	server, err := fileServer(map[string]string{
-		"/cirros.tar.gz": "/cirros.tar.gz",
-	})
+	exportCmd := exec.Command(dockerBinary, "export", cleanedContainerID)
```

Comment:
can we validate that the export is a sane tar file? otherwise we might get to a point where some garbage export imports 'ok'